typescript/vrotsc/e2e/cases/canvas-items/end-item-happy.wf.ts

```diff
@@ -0,0 +1,26 @@
+import { Workflow, In, Out, RootItem, WorkflowEndItem } from "vrotsc-annotations";
+
+@Workflow({
+	name: "Workflow End Happy",
+	path: "VMware/PSCoE",
+	description: "Workflow with root and end item",
+	attributes: {
+		endMode: {
+			type: "number"
+		}
+	}
+})
+export class WorkflowEnd {
+
+	@RootItem()
+	public initiateWorkflow() {
+		// NOOP
+	}
+
+	@WorkflowEndItem({
+		endMode: 0
+	})
+	public workflowEnd(@In endMode: number) {
```

Comment:
Same comment here

Reply:
It is required by the XML representation of the component.